packages/ckeditor5-coremedia-link/src/contentlink/ui/ContentLinkActionsViewExtension.ts

```diff
@@ -5,7 +5,11 @@ import { LinkUI } from "@ckeditor/ckeditor5-link";
 // LinkActionsView: See ckeditor/ckeditor5#12027.
 import LinkActionsView from "@ckeditor/ckeditor5-link/src/ui/linkactionsview";
 import ContentLinkView from "./ContentLinkView";
-import { CONTENT_CKE_MODEL_URI_REGEXP } from "@coremedia/ckeditor5-coremedia-studio-integration/src/content/UriPath";
+import {
+  CONTENT_CKE_MODEL_URI_REGEXP,
+  requireContentUriPath,
+  UriPath,
+} from "@coremedia/ckeditor5-coremedia-studio-integration/src/content/UriPath";
```

Comment:
We may want to change to `index.ts` import here. It was only a shortcut for 37.x update, that I did not apply this change everywhere.